src/components/pages/donation_form/NewsletterOption.vue

```diff
@@ -14,31 +15,22 @@
 
 </template>
 
-<script lang="ts">
-import Vue from 'vue';
-import { NS_ADDRESS } from '@/store/namespaces';
-import { action } from '@/store/util';
-import { setNewsletterChoice } from '@/store/address/actionTypes';
-import FunCheckbox from '@/components/shared/form_inputs/FunCheckbox.vue';
+<script setup lang="ts">
+// TODO: This component should not use the store, but be passed the default value as a prop and emit changes
```

Comment:
is this a leftover todo for this PR or for some other time?

Reply:
No this is a TODO for the future. We need to treat components in the front end similarly to how we treat modules in the back end in that they should have as little knowledge of the system as possible. So for example this component it should take in a model and emit changes, not access the store directly.